### ISSUE TITLE
Variable image sizes

### ISSUE THREAD

**Issue by aaarrmiinnn** [aaarrmiinnn/anomaly_detection_aquatic_data#5]
Images we have are varied on x, y dimension. Their require to be of same size before being fed into the nn.

**Comment by aaarrmiinnn** [comment]
Cannot use:
`tf.image.resize_image_with_crop_or_pad(
    image,
    target_height,
    target_width
)`
because it explicitly requires 3D or 4D images only.

**Comment by aaarrmiinnn** [comment]
On the TFrecords writer side we resize_or_pad all images to one standard size, i.e. **(3208,3387,4)**. The downside is that the function `tf.image.resize_image_with_crop_or_pad` is only available in Tensorflow, which apparently slows down the TFrecords writing significantly. This is the block that is modified to accommodate this change:

` 
image_tf = tf.image.resize_image_with_crop_or_pad(image_tf, target_height=rows_count, target_width=cols_count)
`

**Comment by aaarrmiinnn** [comment]
Yet to be implemented on Validation set.

**Comment by aaarrmiinnn** [comment]
Following the previous comment, now I was able to move the resizing with `image_tf = tf.image.resize_image_with_crop_or_pad(image_tf, target_height=rows_count, target_width=cols_count)` to the reading side. So the writers are not performing any image resizing including padding and cropping.
